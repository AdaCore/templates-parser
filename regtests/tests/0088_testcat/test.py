from test_support import *

gnatmake('testcat')
run('testcat')
