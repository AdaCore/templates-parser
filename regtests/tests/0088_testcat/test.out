testcat template
testcat template
testcat template
